src/main/java/reposense/model/BlurbMap.java

```diff
@@ -0,0 +1,67 @@
+package reposense.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents the mapping between the repo URL to the associated blurb.
+ */
+public class BlurbMap {
+    @JsonProperty("urlBlurbMap")
+    private final Map<String, String> urlBlurbMap;
+
+    public BlurbMap() {
+        this.urlBlurbMap = new HashMap<>();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+
+        if (obj instanceof BlurbMap) {
+            BlurbMap bm = (BlurbMap) obj;
+            return bm.urlBlurbMap.equals(this.urlBlurbMap);
+        }
+
+        return false;
+    }
+
+    /**
+     * Builder class to help in the building of a {@code BlurbMap}.
+     */
+    public static final class Builder {
```

Comment:
I'm not sure that this builder is entirely necessary, given that the class has no constructor arguments and given the somewhat untuitive behavior of our builders currently. Is there something you forsee being added that would benefit from this added complexity?

Reply:
I wanted to use builders in the event that we wanted to add more functionalities in the future (images, etc.), but if that is not necessary then I can refactor to remove the builder class entirely!

Reply:
I assume since we're using a map from identifier -> blurb data, we can add to the instance directly even if we add additional media to the blurb, without needing to provide them upfront in a constructor call. This might make more sense than going through the Builder indirection which is itself single use anyway.

Reply:
Sure thing, will refactor and commit the new changes soon!